benchmarks.project.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Roact Benchmarks Place",
```

Comment:
@jeparlefrancais I liked your idea of keeping benchmarks in a separate place. Updated here.